### PR TITLE
add constrain for wechat logo default ratio

### PR DESCRIPTION
**Pull request by novain1102**
Re: #https://github.com/cocos/3d-tasks/issues/17084

### Changelog

*

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

**Comment by minggo** [comment]
The titles says the constrain is for wechat logo, but the logic is for all minigames.

**Comment by minggo** [comment]
Ok, it is my fault. It is in wechat template.

File: templates/wechatgame/first-screen.ejs [hunk 1]
```diff
@@ -179,9 +179,10 @@ function initProgressVertexBuffer() {
 }
 
 function updateVertexBuffer() {
-    // the ratio value may be adjusted according to the expected image display effect
-    const widthRatio = image.width / canvas.width * 1.35 * displayRatio; 
-    const heightRatio = image.height / canvas.height * 1.35 * displayRatio;
+    // By default, maintain aspect ratio by constraining display at 200px height
+    const defaultRatio = 200 / image.width;
```
Comment (minggo) [inline templates/wechatgame/first-screen.ejs]: Why use 200? It is a magic number.

Comment (novain1102) [reply templates/wechatgame/first-screen.ejs]: 1. cocos default logo size is 140 * 200
image
3. defualt height in splashScreen.ts is also 200
image

**Review by minggo — APPROVED** [review]

**Pull request merged**